### PR TITLE
controller:beerocks:cli:fix bml_conn_map command

### PR DESCRIPTION
**Pull request by CoralMalachi**
This pull request fixes the` bml_conn_map` command in the
beerocks CLI.

The following changes were made :

- Move the action of set to false pThis->pending_response to the right place

- Remove the "-w" option from in the beerocks CLI.

Signed-off-by: Coral Malachi <coral.malachi@intel.com>

### CODE REVIEW

File: controller/src/beerocks/cli/beerocks_cli_main.cpp [hunk 1]
```diff
@@ -486,10 +483,6 @@ int main(int argc, char *argv[])
             cli_role       = CLI_NON_INTERACTIVE;
             break;
         }
-        case 'w': {
```
Comment (morantr) [inline controller/src/beerocks/cli/beerocks_cli_main.cpp:489]: Please add to the commit message why did you remove the wait option.

**Review by morantr — CHANGES_REQUESTED** [review]

File: controller/src/beerocks/cli/beerocks_cli_main.cpp (hunk 1 above)
Comment (arnout) [reply controller/src/beerocks/cli/beerocks_cli_main.cpp:489]: > Please add to the commit message why did you remove the wait option.

Whatever the reason: it should probably be a separate commit anyway.

**Review by arnout — COMMENTED** [review]
Couple of comments about the commit message:

> The following commit moves the action of set to false the

"The following commit" refers to the next commit, not to this one. So it should be "This commit". However, saying "This commit" is usually redundant. As explained in the text about writing a good commit message, you should first explain the problem, then describe the change in the imperative. So the sentence should be "Move the action of set to false `pThis->pending_response` to the right place.", and it should come after the problem description.


> `pThis->pending_response` to the right place.
> 
> In cases the connection map needs to be split to more than one fragment

In case

> nothing is printed, causing a few problems, for example, making the
> AL_MAC be empty.
> 
> The solution is to set  `pThis->pending_response` to true only after we

set to *false*

Also, use the imperative.

> have read the last fragment in the `connection_map_cb` method. then when

then -> Then

> we check if we still need to wait for more fragments at the
> `cli_non_interactive` method we get the right result, prevents us from
> disconnecting at the wrong time.
>

File: controller/src/beerocks/cli/beerocks_cli_main.cpp (hunk 1 above)
Comment (CoralMalachi) [reply controller/src/beerocks/cli/beerocks_cli_main.cpp:489]: Yes, I meant to do this yesterday, I'll do this now.

**Comment by CoralMalachi** [comment]
> Couple of comments about the commit message:
> 
> > The following commit moves the action of set to false the
> 
> "The following commit" refers to the next commit, not to this one. So it should be "This commit". However, saying "This commit" is usually redundant. As explained in the text about writing a good commit message, you should first explain the problem, then describe the change in the imperative. So the sentence should be "Move the action of set to false `pThis->pending_response` to the right place.", and it should come after the problem description.
> 
> > `pThis->pending_response` to the right place.
> > In cases the connection map needs to be split to more than one fragment
> 
> In case
> 
> > nothing is printed, causing a few problems, for example, making the
> > AL_MAC be empty.
> > The solution is to set  `pThis->pending_response` to true only after we
> 
> set to _false_
> 
> Also, use the imperative.
> 
> > have read the last fragment in the `connection_map_cb` method. then when
> 
> then -> Then
> 
> > we check if we still need to wait for more fragments at the
> > `cli_non_interactive` method we get the right result, prevents us from
> > disconnecting at the wrong time.

Ok

File: controller/src/beerocks/cli/beerocks_cli_main.cpp (hunk 1 above)
Comment (CoralMalachi) [reply controller/src/beerocks/cli/beerocks_cli_main.cpp:489]: Fixed.

**Review by morantr — APPROVED** [review]

**Review by arnout — APPROVED** [review]
Very good commit messages, thanks.

**Review by tomereli — APPROVED** [review]
